inspire/modules/deposit/workflows/literature.py

```diff
@@ -506,9 +506,10 @@ def restructure_ffts(fft):
         userid = deposition.user_id
         user = UserInfo(userid)
         email = user.info.get('email', '')
-        external_ids = UserEXT.query.filter_by(id_user=userid).all()
+        external_ids = UserEXT.query.filter_by(id=userid).all()
```

Comment:
Hum but the keys are
```
+---------+------------------+------+-----+---------+-------+
| Field   | Type             | Null | Key | Default | Extra |
+---------+------------------+------+-----+---------+-------+
| id      | varbinary(255)   | NO   | PRI | NULL    |       |
| method  | varchar(50)      | NO   | PRI | NULL    |       |
| id_user | int(15) unsigned | NO   | MUL | NULL    |       |
+---------+------------------+------+-----+---------+-------+
```
Shouldn't we query the `id` key?

We'd have something like
```
mysql> select * from userEXT;
+----+--------+------------------------+
| id | method | id_user                |
+----+--------+------------------------+
| 1  | orcid  |    1234-1234-1234-1234 |
+----+--------+------------------------+
```
Right?